transpile/classes.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/python3
+
+class Block:
+    id = -1
```

Comment:
You're defining all of these as static variables I believe which is not what should be done. They should be member variables.
```python
class Block(object):
    def __init__(self):
        self._id = -1
        # etc.
```

Same goes for the other classes.